extras/installer.sh

```diff
@@ -277,6 +277,9 @@ fi
 
 echo
 read -e -p "Directory to install into: " -i "/opt/plexupdate" FULL_PATH
+if [[ "$FULL_PATH" == *"~"* ]]; then # If path contains a tilde
```

Comment:
I think just `~` is fine. Testing every special character isn't reasonable without using eval, and we're mostly doing this as a courtesy for the user anyway (so they don't get an unexpected directory called `~foo`). If you're any more adventurous than that with your filenames you either know **exactly** what you're doing, or you have **no idea** what you're doing. Either way, not something we can reasonably help you with. 😄